### PR TITLE
Add uri format validation for new "terms" field

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/schemas/space.json [hunk 1]
```diff
@@ -20,6 +20,11 @@
           "title": "about",
           "maxLength": 160
         },
+        "terms": {
+          "type": "string",
+          "title": "terms",
+          "format": "uri"
```
Comment (bonustrack) [inline src/schemas/space.json:26]: Can you add this format on "website" field too?

Comment (samuveth) [reply src/schemas/space.json:26]: `uri` will not be invalid without`*://` at the beginning. Any what these other ones mean?
- uri-reference: URI reference, including full and relative URIs.
- uri-template: URI template according to RFC6570
Source: https://github.com/ajv-validator/ajv-formats

Comment (samuveth) [reply src/schemas/space.json:26]: I don't think this will work because domain field isn't really a `uri`.

Comment (bonustrack) [reply src/schemas/space.json:26]: I mean `website` field, it's another one

Comment (samuveth) [reply src/schemas/space.json:26]: Oh, I see

**Review by bonustrack — APPROVED** [review]

**Pull request merged**